src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorizeSearchIntegrationTest.java

```diff
@@ -872,7 +872,61 @@ public void findOneAndUpdate() {
 
     @Test
     @Order(2)
-    public void updateOne() {
+    public void findOneAndUpdate_updateClause() {
+      insertVectorDocuments();
+      String json =
+          """
+                  {
+                    "findOneAndUpdate": {
+                      "sort" : {"$vectorize" : "A deep learning display that controls your mood"},
+                      "update" : {"$set" : {"status" : "active","$vectorize":"An AI quilt to help you sleep forever"}},
```

Comment:
We should probably also test `$vectorize` being set to the exact same value. Is there a good way to test that vectorize is not called in that case?

Reply:
Thanks, I have added some tests in [DocumentUpdaterTest.java](https://github.com/stargate/data-api/pull/1169/files#diff-504d091a2e59c884abcfc39b63f95a7dd5e223348cc06369e6cd637fafdd9582).
This is to check if there is any diff for $vectorize.